src/components/audio-zone.js

```diff
@@ -42,17 +49,35 @@ AFRAME.registerComponent("audio-zone", {
     this.enableDebug(this.data.debuggable && window.APP.store.state.preferences.showAudioDebugPanel);
   },
 
+  tick: (() => {
+    const quat = new THREE.Quaternion();
+    return function() {
+      this.el.object3D.quaternion.set(0, 0, 0, 1);
+      this.el.object3D.quaternion.multiply(this.el.object3D.parent.getWorldQuaternion(quat).inverse());
+      this.el.object3D.updateMatrix();
+    };
+  })(),
+
   isEnabled() {
     return this.data.enabled;
   },
 
   getBoundingBox: (function() {
     const bbaa = new THREE.Box3();
     return function() {
+      this.debugMesh.geometry.computeBoundingBox();
```

Comment:
Similar to `computeBoundingSphere()`, better to consider to add a guard?

Reply:
Updated.